### DISCUSSION TITLE
Keyball44 ランドが剥がれてしまった

### DISCUSSION THREAD

**Discussion by watagit** [Yowkees/keyball#271] [Q&A]
はんだ付け中に赤枠部分のランドが剥がれてしまい、ダイオードが取り付けられなくなってしまいました。
この場合の対処方法を教えていただけますと幸いです。

![image](https://github.com/Yowkees/keyball/assets/44804976/d8e9d272-5452-4050-a79b-5034e5634fa4)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ひとまず生きているパッドに通常通りダイオードを半田付けしていただき、ダイオードの反対側の端子、緑点どうしをエナメル線等でショートしてください。
これで復活できると思いますので、お手数ですがご対応お願いいたします。

![KB44L_diodefix](https://github.com/Yowkees/keyball/assets/84209772/42ea701d-78f2-4194-a092-ba5c68a40ade)

**Reply by watagit** [reply]
@Yowkees 
ありがとうございます！復活しました 👍

**Comment by Yowkees** [comment]
復活できてよかったです！安心しました。
今後ともどうぞよろしくお願いいたします。
